### DISCUSSION TITLE
Weasis 3D hardware requirements

### DISCUSSION THREAD

**Discussion by AlexZaka** [nroduit/Weasis#407] [Q&A]
What are the Weasis 3D hardware requirements? It seems that my Intel UHD Graphics 630 together with Mesa 21.3.8 are not sufficient. Is there any list of 3D hardware tested and compatible with Weasis?

**Comment by nroduit** [comment]
This is more related to the OpenGL capabilities, see the requirements [here](https://nroduit.github.io/en/tutorials/dicom-3d-viewer).
